gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -96,9 +96,10 @@
   @Hot private final Subject<Boolean> locationLockChangeRequests = PublishSubject.create();
   @Hot private final Subject<CameraUpdate> cameraUpdateSubject = PublishSubject.create();
 
-  /** Polyline drawn by the user but not yet saved as polygon. */
+  /** Short-lived {@link MapFeature} used during add/edit flows. */
   @Hot
-  private final PublishProcessor<PolygonFeature> drawnPolylineFeature = PublishProcessor.create();
+  private final PublishProcessor<ImmutableSet<MapFeature>> ephemeralMapFeatures =
```

Comment:
Trying to think of a more description name than "transient" and "ephemeral" here. Maybe "unsavedMapFeatures"?

Reply:
Done